Source/Operations/AddDefaultConstraintOperation.cs

```diff
@@ -29,19 +22,19 @@ public sealed class AddDefaultConstraintOperation : ConstraintOperation
 
 		public override string ToIdempotentQuery()
 		{
-			return String.Format("if object_id('{0}.{1}', 'D') is null{2}\t{3}", SchemaName, Name, Environment.NewLine, ToQuery());
+			return $"if object_id('{SchemaName}.{Name}', 'D') is null{Environment.NewLine}\t{ToQuery()}";

```

Comment:
Environment.NewLine - great for writing local text files. Not necessary applicable to remote servers (MSSQL) or protocols (HTTP headers).

Reply:
We don't have any tests running on Linux/mac OS yet, so I'm not sure how SQL Server on those operating systems support `\r`. I'm assuming it does.

Reply:
Oh, since these queries do get written out to files by Rivet's `Convert-Migration.ps1` script, I'm using the OSes newline.